container/kvm/container.go

```diff
@@ -19,6 +24,9 @@ type kvmContainer struct {
 	// this allows for checking when we don't know, but using a
 	// value if we already know it (like in the list situation).
 	started *bool
+
+	pathfinder func(string) (string, error)
```

Comment:
renamed. This started out as an internal package interface, but became just a function.